trello/api/features/negative/OrganizationNegative.feature

```diff
@@ -0,0 +1,29 @@
+# Created by Enrique Carrizales at 2/10/2020
+Feature: Organization
+  As a regular user, It manages a Organization.
+
+  @defect
+  Scenario: API can't create a Organization without name
+    When Sets a "POST" request to "/organizations/"
+      | key  | value        |
+      | desc | without Name |
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "Display Name must be at least 1 character"
+    And Sets a "GET" request to "/organizations/organizationObject.id"
+    And Sends request
+    And Should return status code 404
+
+
+  @defect
+  Scenario: API can't create a Organization with a wrong website
```

Comment:
improve this title please, for example to Organization ca't be created with a wrong website addrees